products/php.md

```diff
@@ -22,8 +22,8 @@ releases:
 -   releaseCycle: "8.1"
     support: 2023-11-25
     eol: 2024-11-25
-    latest: "8.1.13"
-    latestReleaseDate: 2022-11-22
+    latest: "8.1.14"
```

Comment:
8.1.14 is not yet listed on https://www.php.net/ChangeLog-8.php or https://www.php.net/downloads.php.